presto-raptor/src/main/java/com/facebook/presto/raptor/systemtables/PreparedStatementBuilder.java

```diff
@@ -84,19 +86,22 @@ private static String getWhereClause(
             List<String> columnNames,
             List<Type> types,
             Set<Integer> uuidColumnIndexes,
-            List<ValueBuffer> bindValues)
+            List<ValueBuffer> bindValues,
+            Optional<String> additionalFixedCondition)
     {
-        if (tupleDomain.isNone()) {
-            return "";
+        ImmutableList.Builder<String> conjunctsBuilder = ImmutableList.builder();
+        if (!tupleDomain.isNone()) {
```

Comment:
This refactoring makes it difficult to see the real change. Please split this out into a separate commit.